src/app/components/list/overview/index.html

```diff
@@ -60,6 +60,120 @@
+      isOptional="true"
+    >
+    Specifies a set of fields to sort by. Acceptable value: <sky-code>Array of ListSortModel</sky-code> properties.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="ListFilterModel properties">
+    <sky-demo-page-property
+      propertyName="name"
+      isOptional="true"
+    >
+    Specifies the name of the filter. This property accepts <sky-code>string</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="label"
+      isOptional="true"
+    >
+    Specifies the text for the filter's summary item. If not specified, then the filter summary item text defaults to the <sky-code>value</sky-code> of the filter. This property accepts <sky-code>string</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="dismissable"
+      isOptional="true"
+    >
+    Indicates whether users can dismiss the filter's summary item. This property accepts <sky-code>boolean</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="value"
+      isOptional="true"
+    >
+    Specifies the current value of the filter. This property accepts <sky-code>any</sky-code> value.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="defaultValue"
+      isOptional="true"
+    >
+    Specifies the default value of the filter. If the value of the filter is set to the default value, then the filter is not applied.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="filterFunction"
+      isOptional="true"
+    >
+    Specifies the function to apply to determine whether an item is filtered. This property is required when using in memory data provider. Acceptable values: <sky-code>(item: ListItemModel, filter: any) => boolean</sky-code>. This property accepts <sky-code>boolean</sky-code> values.    
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="ListSearchModel properties">
+    <sky-demo-page-property
+      propertyName="searchText"
+      isOptional="true"
+    >
+    Specifies the text to search. This property accepts <sky-code>string</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="functions"
+      isOptional="true"
+    >
+    Specifies an array of functions that should return a <sky-code>boolean</sky-code> value of <sky-code>true</sky-code> when the search is successful. Acceptable values: <sky-code>(data: any, searchText: string) => boolean></sky-code>.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="fieldSelectors"
+      isOptional="true"
+    >
+    Specifies an array of <sky-code>string</sky-code> types which run against search fields. The <sky-code>string</sky-code> values should match the data's <sky-code>field</sky-code> value.
```

Comment:
I'm not following this description. What does it mean to run types against search fields? And what `field` values are we talking about? We format `field` as a code snippet, which suggests were are referring to a specific property, but I don't see any `field` property anywhere else on this page. I suspect this description should be something like "Specifies the types to search. The types correspond to `field` values that you specify with the `data` property. This property accepts an array of `string` values." (where "the `data` property" could link to the List properties section.) But that said, I'm really not entirely sure if my understanding is correct here, and we need to sort this out with a dev.

Reply:
Agree with @blackbaud-johnly , but have one thought -
Should it be "columns" instead of "types"? I'm not sure what "type" refers to in this context.

Reply:
Thanks, Alex. I should have asked what we meant by "types." I think subbing out "types" for "columns" in my suggested edit makes sense. ... Denise, let me know if I can do anything else here.